syntax = "proto2";

package KeyActionPb;

enum KEY_ACTION
{
    KEY_ACTION_LAST_COMMON = 100;
    // This enumeration starts at 101 for Professor-specific actions. Values under 100 are copied from ActionCommon_t in CommonIntents.h
    KEY_ACTION_POWER = 101;
    KEY_ACTION_BLUETOOTH_CAROUSEL_DISCOVERABLE_CONNECT_TO_LAST = 20;
    KEY_ACTION_BLUETOOTH_SEND_TO_DISCOVERABLE = 21;
    KEY_ACTION_BLUETOOTH_CLEAR_PAIRING_LIST = 22;
    KEY_ACTION_PRESET_STORE_1 = 27;
    KEY_ACTION_PRESET_STORE_2 = 28;
    KEY_ACTION_PRESET_STORE_3 = 29;
    KEY_ACTION_PRESET_STORE_4 = 30;
    KEY_ACTION_PRESET_STORE_5 = 31;
    KEY_ACTION_PRESET_STORE_6 = 32;
    KEY_ACTION_PRESET_SELECT_1 = 33;
    KEY_ACTION_PRESET_SELECT_2 = 34;
    KEY_ACTION_PRESET_SELECT_3 = 35;
    KEY_ACTION_PRESET_SELECT_4 = 36;
    KEY_ACTION_PRESET_SELECT_5 = 37;
    KEY_ACTION_PRESET_SELECT_6 = 38;
    KEY_ACTION_VOLUME_UP = 109;
    KEY_ACTION_VOLUME_DOWN = 110;
    KEY_ACTION_PLAY_PAUSE = 1;
    KEY_ACTION_SKIP_FORWARD = 2;
    KEY_ACTION_SKIP_BACK = 3;
    KEY_ACTION_MUTE = 114;
    KEY_ACTION_SOUNDTOUCH = 115;
    KEY_ACTION_CONNECT = 116;
    KEY_ACTION_ACTION = 117; // @TODO PGC-449 VOICE_CAROUSEL
    KEY_ACTION_TV = 118;
    KEY_ACTION_THUMB_UP = 119;
    KEY_ACTION_THUMB_DOWN = 120;
    KEY_ACTION_FACTORY_DEFAULT = 8;
    KEY_ACTION_WIFI_OFF = 7;
    KEY_ACTION_AP_SETUP = 5;
    KEY_ACTION_PAIR_SPEAKERS = 124;
}

